kickstarts/partials/packages/excludes.ks.erb

```diff
@@ -1,33 +1,18 @@
-# exclude these defaults from the @core group, with a few (commented) exceptions
--aic94xx-firmware
--atmel-firmware
--b43-openfwwf
--bfa-firmware
-#efibootmgr
-#grub
--ipw2100-firmware
--ipw2200-firmware
--ivtv-firmware
--iwl100-firmware
--iwl1000-firmware
--iwl3945-firmware
--iwl4965-firmware
--iwl5000-firmware
--iwl5150-firmware
--iwl6000-firmware
--iwl6000g2a-firmware
--iwl6050-firmware
-#kernel-firmware
--libertas-usb8388-firmware
-#postfix
--ql2100-firmware
--ql2200-firmware
--ql23xx-firmware
--ql2400-firmware
--ql2500-firmware
--rt61pci-firmware
--rt73usb-firmware
--xorg-x11-drv-ati-firmware
--zd1211-firmware
+# Excluded packages.
+
+# ALSA is not needed. We don't need to play sounds in our appliance.
+-alsa-*
+
+# We don't need any firmware files. We always run as a virtual appliance, and
+# none of the virtual hardware devices implement by any of the major hypervisor
+# vendors needs firmware.
+-*-firmware
+
+# NetworkManager is not needed. The appliance uses the "traditional" way of
+# networking.
+-NetworkManager*
```

Comment:
installed but disabled.  some customers may enable it to do realm discovery for AD domains before joining them (external auth AD).